src/app.ts

```diff
@@ -29,4 +30,9 @@ app.listen(3000, () => {
     console.log('Server started on port 3000');
 });
 
-app.get('/', getAllDatabases);
\ No newline at end of file
+
+app.get('/deliveryEmployeeForm', getCreateDeliveryEmployeeForm);
+
+app.post('/deliveryEmployeeForm', postDeliveryEmployeeForm);
+
+app.get('/deliveryEmployee');
```

Comment:
Is this needed?

Reply:
Thomas has resolved this issue, getting rid of unused line 38